### PR TITLE
Zc/feature/config docs

### PR DESCRIPTION
**Pull request by zcericola**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -1685,7 +1685,13 @@ This endpoint updates a location for your business.
 
 ### HTTP Request
 
-`PUT https://platform.veryableops.com/api/profile/locations`
+`PUT https://platform.veryableops.com/api/profile/locations/:locationId`
```
Comment (andrewplan) [inline source/index.html.md]: Wrap this in brackets instead, e.g. `<locationId>`

File: source/index.html.md [hunk 2]
```diff
@@ -1966,8 +1974,873 @@ operatorId | number | yes | The operatorId for the operator you want to toggle.
...
+```shell
+curl -X PUT "http://localhost:3000/api/workcenters/1/?=businessId=300"
+    -H "Authorization: Bearer [JWT string]"
+```
+
+This endpoint gets the work centers for your business.
+
+### HTTP Request
+
+`PUT https://platform.veryableops.com/workcenters/:workcenterId`
```
Comment (andrewplan) [inline source/index.html.md]: For all HTTP endpoint definitions, anything using `:` to denote a URL param, please use `<brackets>` instead.

File: source/index.html.md (hunk 1 above)
Comment (zcericola) [reply source/index.html.md]: done

File: source/index.html.md (hunk 2 above)
Comment (zcericola) [reply source/index.html.md]: done

File: source/index.html.md [hunk 3]
```diff
@@ -2443,7 +2443,7 @@ This endpoint creates a production schedule for your business.
 
 ### HTTP Request
 
-`DELETE https://platform.veryableops.com/schedules/:scheduleId`
+`DELETE https://platform.veryableops.com/schedules<scheduleId>`
```
Comment (andrewplan) [inline source/index.html.md]: Please put a `/` between `schedules` and `<scheduleId>`.

File: source/index.html.md [hunk 4]
```diff
@@ -2766,7 +2766,7 @@ This endpoint creates a SKU for a business.
 
 ### HTTP Request
 
-`POST https://platform.veryableops.com/skus/:skuId`
+`POST https://platform.veryableops.com/skus<skuId>`
```
Comment (andrewplan) [inline source/index.html.md]: Please put a `/` between `skus` and `<skuId>`.

**Pull request merged**